packages/@guardian/source-foundations/src/index.ts

```diff
@@ -75,6 +87,28 @@ export const palette = {
 	focus,
 };
 
+// theme colours
+export const defaultTheme = {
+	background,
+	border,
+	line,
+	text,
+};
+
+export const brandTheme = {
+	brandBackground,
+	brandBorder,
+	brandLine,
+	brandText,
+};
+
+export const brandAltTheme = {
+	brandAltBackground,
+	brandAltBorder,
+	brandAltLine,
+	brandAltText,
+};
```

Comment:
Any strong feelings on whether this `theme` stuff also belongs on the `palette` object? My thoughts are: they do belong.